controllers/controllers.js

```diff
@@ -57,6 +58,15 @@ exports.postNewCommentById = (req, res, next) => {
     .catch(next);
 };
 
+exports.deleteCommentById = (req, res, next) => {
+    const {comment_id} = req.params
+    deleteComment(comment_id)
+    .then((result) => {
+        res.status(204).send({result})
```

Comment:
Express won't send back a response here so you would only want either `res.status(204)` or `res.status(204).send()`